scrapy/contrib/loader/processor.py

```diff
@@ -70,3 +74,254 @@ def __init__(self, separator=u' '):
 
     def __call__(self, values):
         return self.separator.join(values)
+
+
+class Strip(object):
+
+    def __init__(self, chars=None):
```

Comment:
What about `lstrip` and `rstrip`? Can we have a `mode` flag accepting 'r'/'right', 'l'/'left' or None. Store `self.strip_func = <stripfunc>`. 
We should also support `chars`.